authors/apps/authentication/serializers.py

```diff
@@ -136,10 +137,27 @@ class UserSerializer(serializers.ModelSerializer):
         min_length=8,
         write_only=True
     )
+    # When a field should be handled as a serializer, we must explicitly say
+    # so. Moreover, `UserSerializer` should never expose profile information,
+    # so we set `write_only=True`.
+    profile = ProfileSerializer(write_only=True)
+
+    # We want to get the `bio` and `image` fields from the related Profile
+    # model.
+    bio = serializers.CharField(source='profile.bio', read_only=True)
+    image = serializers.CharField(source='profile.image', read_only=True)
```

Comment:
why have you considered a charField for the image other than a urlfield or an image field?

Reply:
@Emmanuelaaron thanks for noticing this out. Let me rectify this accordingly.

Reply:
@Emmanuelaaron please note that this was resolved. kindly review and confirm.